test/controllers/tasks/active.psc.details.controller.unit.ts

```diff
@@ -67,5 +70,16 @@ describe("Active psc details controller tests", () => {
       expect(response.text).toContain("Charity - Unincorporated Association");
       expect(response.text).toContain("Ownership of voting rights - more than 75%");
     });
+
+    it("should navigate to psc statement page if no psc is found", async () => {
```

Comment:
also do a similar test that returns undefined from mockGetPscs.mockResolvedValueOnce
to test that it goes to the psc statement page as well (same as the test when [] gets returned)